app/views/hyrax/base/_form_progress.html.erb

```diff
@@ -0,0 +1,66 @@
+<aside id="form-progress" class="form-progress card">
```

Comment:
Did we override a specific line in this file? If so, could you add at the top that this is a Hyrax override with the Hyrax version specified, as well as the lines where the override starts